Sources/Sentry/SentryUIApplication.m

```diff
@@ -0,0 +1,43 @@
+#import "SentryUIApplication.h"
+
+#if SENTRY_HAS_UIKIT
+
+@implementation SentryUIApplication
+
++ (UIApplication *)sharedApplication
+{
+    if (![UIApplication respondsToSelector:@selector(sharedApplication)])
+        return nil;
+
+    return [UIApplication performSelector:@selector(sharedApplication)];
+}
+
++ (NSArray<UIWindow *> *)windows
+{
+    UIApplication *app = SentryUIApplication.sharedApplication;
+    if (app == nil)
+        return nil;
+
+    NSMutableArray *result = [NSMutableArray new];
+
+    if (@available(iOS 13.0, *)) {
+        if ([app respondsToSelector:@selector(connectedScenes)]) {
+            for (UIScene *scene in app.connectedScenes) {
+                if (scene.activationState == UISceneActivationStateForegroundActive
+                    && scene.delegate && [scene.delegate respondsToSelector:@selector(window)]) {
+                    [result addObject:[scene.delegate performSelector:@selector(window)]];
+                }
+            }
+        }
+    }
+
+    if ([app.delegate respondsToSelector:@selector(window)]) {
+        [result addObject:app.delegate.window];
```

Comment:
`m`: We also add windows above. Are we sure we are not going to add windows twice?

Reply:
It might have more than one. Although unlikely. And that's okay.